### PR TITLE
Code review for LIST method

### PR DESCRIPTION
**Pull request by IrregularHunterX**
Unit test results are below

Test Results
Test Results again

I mentioned this on Slack already, but I did not include code to return a list that filters on:

1. The type of item, I don't think the DB model has type anymore
2. Quantity. I realized that it may be better for us to specify a keyword like "RESTOCK" that will return a list of items where quantity < restock level. I can definitely add this in the next sprint if it's sort of too late now. What does everyone else think?

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from flask import Flask, jsonify, request, url_for, make_response, abort
 from service.common import status  # HTTP Status Codes
 from service.models import Inventory
```
Comment (vkrishna1997) [inline service/routes.py:9]: Slight nitpick but can we change this to:
```suggestion
from service.models import Inventory, Condition
```

File: service/routes.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
...
+from service.models import Condition
```
Comment (vkrishna1997) [inline service/routes.py:10]: Same suggestion as above:
```suggestion
```

**Comment by vkrishna1997** [comment]
Looks good to me!
As for your point about RESTOCK levels, I think that's a good idea. Would be great to have in Sprint 2 potentially

**Review by vkrishna1997 — APPROVED** [review]

File: service/routes.py (hunk 1 above)
Comment (IrregularHunterX) [reply service/routes.py:9]: Ok, I can make this change in the next sprint

File: service/routes.py (hunk 2 above)
Comment (IrregularHunterX) [reply service/routes.py:10]: Ditto

**Review by MarvinTellis — APPROVED** [review]

**Review by mmclslwsl — APPROVED** [review]

**Pull request merged**